### PR TITLE
Fix signature change from InternalItemsQuery

### PR DESCRIPTION
**Pull request by Stampede10343**
The type changed from ValueTuple to IReadOnlyList, this fixes the syntax
change and also points to the new 10.5-pre1 package to get the api
changes.

### CODE REVIEW

**Comment by Stampede10343** [comment]
This will address Issue #6

File: Jellyfin.Plugin.TMDbBoxSets/Jellyfin.Plugin.TMDbBoxSets.csproj [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>netstandard2.1</TargetFramework>
     <IsPackable>true</IsPackable>
     <AssemblyVersion>4.0.0</AssemblyVersion>
     <FileVersion>4.0.0</FileVersion>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Jellyfin.Controller" Version="10.*" />
+    <PackageReference Include="Jellyfin.Controller" Version="10.5.0-pre1" />
```
Comment (Stampede10343) [inline Jellyfin.Plugin.TMDbBoxSets/Jellyfin.Plugin.TMDbBoxSets.csproj]: Not sure if we need to wait for a proper release to update this, I couldn't get `10.5.*` to work for a version number.

**Comment by Stampede10343** [comment]
I also created a super simple Dockerfile and script for building this according to the README, would we be interested in including that?

File: Jellyfin.Plugin.TMDbBoxSets/Jellyfin.Plugin.TMDbBoxSets.csproj (hunk 1 above)
Comment (joshuaboniface) [reply Jellyfin.Plugin.TMDbBoxSets/Jellyfin.Plugin.TMDbBoxSets.csproj]: Try just 10.*, that should work and is what we did for 10.4.0.

Comment (Stampede10343) [reply Jellyfin.Plugin.TMDbBoxSets/Jellyfin.Plugin.TMDbBoxSets.csproj]: Yeah that's what I tried initially but it seemed to keep pulling 10.4.0, I couldn't get it to pull 10.5 until I specified the full version. It's possible there's a better way but a quick glance at the docs didn't show anything and I checked a couple other plugin implementations and they had the same 10.*

**Comment by Stampede10343** [comment]
Should the version number be bumped? `5.0.0`?

**Comment by mcarlton00** [comment]
version bump and release is generally handled in a separate PR.  helps keep things more granular and finding history that way

File: Jellyfin.Plugin.TMDbBoxSets/Jellyfin.Plugin.TMDbBoxSets.csproj (hunk 1 above)
Comment (Stampede10343) [reply Jellyfin.Plugin.TMDbBoxSets/Jellyfin.Plugin.TMDbBoxSets.csproj]: 10.* works now that 10.5 is in NuGet! I reverted this change

**Comment by dkanada** [comment]
I feel like it's fine to bump in this pull request as well, it's such a minor change.

**Comment by Stampede10343** [comment]
> I feel like it's fine to bump in this pull request as well, it's such a minor change.

Bump to `5.0.0`?

File: Jellyfin.Plugin.TMDbBoxSets/TMDbBoxSetManager.cs [hunk 2]
```diff
@@ -74,8 +74,7 @@ private IReadOnlyCollection<Movie> GetMoviesFromLibrary()
             {
                 IncludeItemTypes = new[] {typeof(Movie).Name},
                 IsVirtualItem = false,
-                OrderBy = new[]
-                {
+                OrderBy = new List<ValueTuple<string, SortOrder>> {
```
Comment (dkanada) [inline Jellyfin.Plugin.TMDbBoxSets/TMDbBoxSetManager.cs]:
```suggestion
                OrderBy = new List<ValueTuple<string, SortOrder>>
                {
```

Comment (dkanada) [reply Jellyfin.Plugin.TMDbBoxSets/TMDbBoxSetManager.cs]: This is how most of the code styles brackets.

**Comment by dkanada** [comment]
yup version++

File: Jellyfin.Plugin.TMDbBoxSets/Jellyfin.Plugin.TMDbBoxSets.csproj [hunk 3]
```diff
@@ -1,10 +1,10 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>netstandard2.1</TargetFramework>
     <IsPackable>true</IsPackable>
-    <AssemblyVersion>4.0.0</AssemblyVersion>
-    <FileVersion>4.0.0</FileVersion>
+    <AssemblyVersion>5.0.0</AssemblyVersion>
+    <FileVersion>5.0.0</FileVersion>
```
Comment (dkanada) [inline Jellyfin.Plugin.TMDbBoxSets/Jellyfin.Plugin.TMDbBoxSets.csproj]: You should also increment the number in `build.yaml` and update the minimum required server version.

**Pull request merged**